service/src/main/java/uk/gov/hmcts/reform/fpl/model/Order.java

```diff
@@ -1,23 +1,37 @@
 package uk.gov.hmcts.reform.fpl.model;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonInclude;
 import lombok.Builder;
 import lombok.Data;
 import uk.gov.hmcts.reform.fpl.enums.OrderStatus;
 import uk.gov.hmcts.reform.fpl.model.common.DocumentReference;
 import uk.gov.hmcts.reform.fpl.model.common.Element;
 import uk.gov.hmcts.reform.fpl.model.common.JudgeAndLegalAdvisor;
+import uk.gov.hmcts.reform.fpl.model.interfaces.IssuableOrder;
 
 import java.util.List;
 
+import static uk.gov.hmcts.reform.fpl.enums.OrderStatus.SEALED;
+
 @Data
 @Builder(toBuilder = true)
 @JsonInclude(JsonInclude.Include.NON_NULL)
-public class Order {
+public class Order implements IssuableOrder {
     private final String hearingDate;
     private final List<Element<Direction>> directions;
     private final OrderStatus orderStatus;
     private final DocumentReference orderDoc;
     private final JudgeAndLegalAdvisor judgeAndLegalAdvisor;
     private final String dateOfIssue;
+
+    @JsonIgnore
+    public boolean isSealed() {
+        return SEALED == orderStatus;
+    }
+
+    @JsonIgnore
+    public boolean isDraft() {
+        return !isSealed();
```

Comment:
I don't think in the current implementation allows for it but this would mean that something that an order with null is a draft. Is this intentional? Also just for consistencies sake here you are doing `draft = !sealed` whereas in the CMO you are doing `sealed = !draft` maybe it would be nicer if it was always one way round. If that is done there could be a default implementation in the interface which just does `!isDraft/Sealed()`